tap_xenon.tex

```diff
@@ -0,0 +1,418 @@
+
+\author{Andrei Rykhlevskii, Daniel O'grady, Tomasz Kozlowski, Kathryn Huff}
+
+\institute{
+        Department of Nuclear, Plasma, and Radiological Engineering, University 
+        of Illinois at Urbana-Champaign \break
+        Urbana, IL
+}
+
+\email{andreir2@illinois.edu}
+
+%%%% packages and definitions (optional)
+\usepackage{graphicx} % allows inclusion of graphics
+\usepackage{caption}  % allows center figures caption
+\usepackage{booktabs} % nice rules (thick lines) for tables
+\usepackage{microtype} % improves typography for PDF
+\usepackage[section]{placeins}
+
+\usepackage[acronym,toc]{glossaries}  % acronyms inclusion
+\include{acros}
+\usepackage{multirow} 
+\usepackage{tabularx}
+\newcolumntype{b}{>{\hsize=1.0\hsize}X}
+\newcolumntype{s}{>{\hsize=.5\hsize}X}
+\newcolumntype{m}{>{\hsize=.75\hsize}X}
+\newcolumntype{x}{>{\hsize=.25\hsize}X}
+\newcolumntype{a}{>{\hsize=.11\hsize}X}
+
+\makeglossaries
+
+\graphicspath{{figures/}}
+
+\newcommand{\SN}{S$_N$}
+\renewcommand{\vec}[1]{\bm{#1}} %vector is bold italic
+\newcommand{\vd}{\bm{\cdot}} % slightly bold vector dot
+\newcommand{\grad}{\vec{\nabla}} % gradient
+\newcommand{\ud}{\mathop{}\!\mathrm{d}} % upright derivative symbol
+
+\begin{document}
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\section{Introduction}
+First-of-kind civil \gls{MSR} was developed, built and operated in \gls{ORNL}  
+in the 	1960s. It was called \gls{MSRE} and at the moment is the
+only one operated 
+\gls{MSR} worldwide. Based on experience from this experiment, the first 
+commercial \gls{MSBR} was designed in early the 1970s. In the thermal spectrum 
+\gls{MSR}, fluorides of fissile and/or fertile materials (i.e., UF$_4$ and/or 
+ThF$_4$) are mixed with carrier salts (i.e., LiF) to form a liquid fuel
+which 
+is circulated in a loop-type primary circuit and leads to following 
+advantages over traditional solid-fueled reactors: (1) low pressure 
+in the primary loop; (2) strong negative thermal feedback; (3) passive decay 
+heat cooling; (4) reduced fuel fabrication costs; (5) online refueling and 
+reprocessing \cite{haubenreich_experience_1970}.
+
+Nevertheless, cost-competitiveness of such innovative designs in the current 
+domestic energy market may only be feasible with load-following operation. 
+Load following operation has the potential to increase the commercial 
+competitiveness of nuclear power dramatically. Due to increasing penetration of 
+renewables on the electric grid, base-load operation carries the risk of 
+correspondingly frequent negative electric energy pricing. Thus, 
+responsiveness to net electricity demand is essential to market relevance for
+new designs \cite{energy_information_administration_u.s._2016}.
+
+The main physical effect that limits the possibilities of power variations in a 
+conventional \gls{LWR} is fission product poisoning, especially the xenon 
+effect (a few hours after the change in the reactor power level).  The 
```

Comment:
I do not think this is true, at least not for PWRs. The way this is written it implies that LWRs do not follow load because they will get poisoned and not have enough positive reactivity to return to full power/remain critical. The limitation is the stresses on the fuel. I think the emphasis of this paragraph should be that unlike LWRs, which are loaded with enough excess reactivity to compensate for this poisoning, the MSR does not have excess reactivity loaded into the core

Reply:
I think your point would be correct at the end of cycle, when there is not a lot of excess reactivity left in the core.

Reply:
Agree with your point but will not change it in the summary. Instead, I will clarify it in my presentation.